lib/sufia/export/version_converter.rb

```diff
@@ -0,0 +1,35 @@
+module Sufia
+  module Export
+    class VersionConverter
+      attr_accessor :uri, :label, :created
+
+      def initialize(uri, version_graph)
+        @uri = uri
+        @created = find_created(uri, version_graph)
+        @label = find_label(uri, version_graph)
+      end
+
+      def to_json(pretty = false)
+        return super unless pretty
+        JSON.pretty_generate(JSON.parse(to_json))
+      end
+
+      private
+
+        def find_created(uri, graph)
```

Comment:
shouldn't need the parameters if there are the same as `@uri` and `@version_graph`